packages/lib-classifier/src/store/annotations/DrawingAnnotation.js

```diff
@@ -0,0 +1,22 @@
+import { types } from 'mobx-state-tree'
+import Annotation from './Annotation'
+import { PointAnnotation } from './DrawingToolAnnotations'
+
+const Drawing = types.model('Drawing', {
+  value: types.array(Object.assign(
+    types.frozen({
+      frame: types.number,
+      toolIndex: types.number
```

Comment:
This is just `tool` in PFE, so we should keep using that for consistency with post-classification analysis unless we have a reason to change it.